<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeroEditor</class>
 <widget class="QDialog" name="HeroEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Die Helden</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_helden"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_held_neu">
             <property name="text">
              <string>Held neu</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_held_entfernen">
             <property name="text">
              <string>Helden entfern</string>
             </property>
             <property name="autoRepeatDelay">
              <number>300</number>
             </property>
             <property name="autoRepeatInterval">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Maximum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Name des Helden</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_name">
           <property name="maxLength">
            <number>25</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_gesicht">
           <property name="sceneRect">
            <rectf>
             <x>0.000000000000000</x>
             <y>0.000000000000000</y>
             <width>0.000000000000000</width>
             <height>0.000000000000000</height>
            </rectf>
           </property>
           <property name="resizeAnchor">
            <enum>QGraphicsView::NoAnchor</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="gesicht">
           <property name="text">
            <string>Gesicht</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>250</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="Koerper">
           <property name="text">
            <string>KÃ¶rper</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_koerper">
           <property name="renderHints">
            <set>QPainter::TextAntialiasing</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_leben">
               <property name="text">
                <string>Leben</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_mana">
               <property name="text">
                <string>Mana</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_kraft">
               <property name="text">
                <string>Kraft</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_ausdauer">
               <property name="text">
                <string>Ausdauer</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_intelligenz">
               <property name="text">
                <string>Intelligenz</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QLineEdit" name="lineEdit_Leben">
               <property name="maxLength">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_Mana">
               <property name="maxLength">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_Kraft">
               <property name="maxLength">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_ausdauer">
               <property name="maxLength">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_Intelligenz">
               <property name="maxLength">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_4">
               <property name="maxLength">
                <number>10</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>45</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>245</width>
             <height>10</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>245</width>
             <height>10</height>
            </size>
           </property>
           <property name="text">
            <string>Schutz vor in %</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="checkBox_eis">
               <property name="text">
                <string>Eis</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_feuer">
               <property name="text">
                <string>Feuer</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_wind">
               <property name="text">
                <string>Wind</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_gift">
               <property name="text">
                <string>Gift</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_betaeubt">
               <property name="text">
                <string>Betaeubt</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QSpinBox" name="spinBox_eis">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_feuer">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_wind">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_gift">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_betaeubt">
               <property name="maximum">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okay">
         <property name="text">
          <string>Okay und Sichern</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Abbrechen</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>HeroEditor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1040</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>986</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
